arch/risc-v/src/common/riscv_exception_common.S

```diff
@@ -99,15 +101,14 @@ exception_common:
   csrr s0, mhartid
 
   /* Switch to interrupt stack */
-
-  bnez s0, 1f
-  la   sp, g_intstacktop
-  j    2f
-1:
-  la   sp, g_intstacktop
-  li   t0, -(CONFIG_ARCH_INTERRUPTSTACK & ~15)
-  add  sp, sp, t0
-2:
+#if CONFIG_N_IRQ_STACKS > 1
+  li t0, ((CONFIG_ARCH_INTERRUPTSTACK) & ~15)
```

Comment:
```suggestion
  li t0, (CONFIG_ARCH_INTERRUPTSTACK & ~15)
```

Reply:
better have parenthesis, just in case someone goes and types "CONFIG_ARCH_INTERRUPTSTACK=2048+1024" or such in his .config file.

Reply:
Ok, removed the parenthesis. They are missing in other places as well, and the syntax for having something else than a single integer wouldn't work for defconfig file anyways (even though for .config it would work)